cllc-public-app/ClientApp/src/app/models/sep-application.model.ts

```diff
@@ -1,7 +1,8 @@
 import { SepLocation } from "./sep-location.model";
 
 export class SepApplication {
-    id: number
+    id?: number;  // indexed db primary key
+    specialEventId: string; // server side primary key
```

Comment:
No change required here, just pointing out the foreign key is a string here, as it should be earlier